server/lib/schema/index.ts

```diff
@@ -289,6 +290,46 @@ const schema = new GraphQLSchema({
           });
         }
       },
+      updateWorkspaceIsPublic: {
```

Comment:
Are you sure about not just having a ``updateWorkspace`` mutation, that accepts multiple optional arguments? I generally recommend staying closer to REST where applicable.

Reply:
the "separate refactor" argument is a fine one.